sonarlint

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-sonarlint -Dsonar.java.libraries=build/libs --src "{src/main/**,email-client/src/main/**}"  --tests "{src/test/**,src/apiTest/**,email-client/src/test/**}"
+sonarlint -Dsonar.java.libraries=build/libs --src "{src/main/**,email-client/src/main/**,ccd-adapter/src/main/**}"  --tests "{src/test/**,src/apiTest/**,email-client/src/test/**,ccd-adapter/src/test/**}"
```

Comment:
this is a script for running it locally, it won't fix missing coverage on the server